testsuites/syncgateway/functional/tests/test_webhooks.py

```diff
@@ -86,7 +86,6 @@ def test_webhooks(params_from_base_test_setup, sg_conf_name, num_users, num_chan
     assert expected_events == received_events
 
 
-@pytest.mark.sanity
```

Comment:
One webhooks test in sanity will be good

Reply:
There is , test_webhooks